netci.groovy

```diff
@@ -29,7 +29,7 @@ def project = GithubProject
             case 'Windows_NT':
                 newJob.with{
                     steps{
-                        batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" && RebuildWithLocalMSBuild.cmd")
+                        batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" && RebuildWithLocalMSBuild.cmd /p:LocalizedBuild=true")
```

Comment:
Thought about that, but it means whenever you run RebuildWithLocalMSBuild on your local machine, you might get dirty xlf files if any resource changed. This might lead to people committing xlf file changes while the loc team is working on them as well.